pages/data-cleanup.tsx

```diff
@@ -700,6 +718,7 @@ export default function DataCleanupPage(props) {
                     <PhoneFields
                       phones={formData.phones}
                       handleArrChange={handleArrChange}
+                      handlePhoneChange={handlePhoneChange}
```

Comment:
Since phone has it's own handleChange does it still need `handleArrChange` passed in as a prop?